ui/ducks/confirm-alerts/confirm-alerts.ts

```diff
@@ -14,6 +14,12 @@ export type Alert = {
    */
   alertDetails?: string[];
 
+  /**
+   * Whether the alert is a blocker and un-acknowledgeable, preventing
+   * the user from proceeding and relying on actions to proceed.
```

Comment:
Should we also clarify the default is `false` if not specified?

Reply:
Done